README.md

```diff
@@ -131,7 +131,7 @@ A choice of web browsers can be configured, though Chrome or Chrome headless are
 
    - In one terminal session, from the project root : 
 
-       ```npm run start-feature```
+       ```npm run start-feature --env=feature.env```
```

Comment:
Is this doc change right? Should it be `npm start-feature-no-webpack --env=feature.env`

Reply:
well running the tests locally with no webpack is a pain since it requires a rebuild everytime anything changes.  So docs are fine in that people would normally prefer not to have to rebuild each time.